spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -185,7 +185,7 @@ definitions:
                       - 2: Differential GNSS (DGNSS)
                       - 3: Float RTK
                       - 4: Fixed RTK
-
+                      - 5: SBAS Position
```

Comment:
This should probably also have 5 as either Dead reckoning or reserved value so that the same enum can be used here and in the other messages?

Reply:
Yeah, should keep values the same across messages. Can call `5` undefined here.

Reply:
agreed, lets leave 5 reserved